_layouts/partners.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+---
+<center class="partner-heading">
+    <h1>Our Partners</h1>
+</center>
+
+<section class='partners'>
+  {% include partners-list.html %}
+</section>
+<center class="partner-heading">
+    <h1>Hopefully Soon</h1>
```

Comment:
Please remove this list from the page. I know @tangollama included those on his list in the issue, but we shouldn't display orgs that aren't actually partners on this page.